ACISobs.py

```diff
@@ -608,8 +610,8 @@ def hrc_science_obs_filter(self, obsidinterval_list):
         """
         acis_and_cti_only = []
         for eachobservation in obsidinterval_list:
-            if  ( (eachobservation[self.in_focal_plane] == "ACIS-I") or \
-                  (eachobservation[self.in_focal_plane] == "ACIS-S") or \
+            if  ( ((eachobservation[self.in_focal_plane] == "ACIS-I") or \
```

Comment:
The extra parens are not needed.  Use of trailing `\` for line continuation is deprecated in modern Python, parens are preferred.